src/Formatter/Result.php

```diff
@@ -60,8 +61,9 @@ public function size()
     }
 
     /**
-     * @return RecordView
      * @throws \RuntimeException When there is no record
+     *
+     * @return RecordView
```

Comment:
I was under the impression that `@throws` should be last in the doc block. But I was wrong.. https://www.phpdoc.org/docs/latest/references/phpdoc/tags/throws.html